src/lib/constants/__init__.py

```diff
@@ -1,8 +1,13 @@
-from src.lib.constants import (
+from lib.constants import (
     browser,
+    element,
     file,
```

Comment:
I don't think you need "w" "r" "a" characters in constants, In this case it's bad because file import clashes with the file keyword in python, which it should not do.